locale/en.po

```diff
@@ -1082,6 +1081,10 @@ msgstr ""
 msgid "METRICS"
 msgstr "Performance Metrics"
 
+#: ../src/engine/PerformanceMetrics.tscn:66
+msgid "FRAME_DELTA"
+msgstr "Delta: {0}"
+
```

Comment:
There's still the previous translation key here, is this intended?

Reply:
That's unintended. I'll clear this from the scene file.